### DISCUSSION TITLE
Using Alexa to start mowing on Luba-1

### DISCUSSION THREAD

**Discussion by Strega-Technologies** [mikey0000/Mammotion-HA#145] [Q&A] [answered]
Luba-1 does not support Alexa, however, some entities of this integration (eg "undocking") can be exposed to Alexa and can be triggered by a voice command. How can I start mowing a  particular area using this integration from an Alexa Voice command ? Thank you.

**Comment by mikey0000** [comment, marked as answer]
Use the service to create a helper or button entity. I'll have a play around myself too.
